man/systemd.netdev.xml

```diff
@@ -504,6 +504,15 @@
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><varname>DiffServiceCodePoint=</varname></term>
+        <listitem>
+          <para>A boolean. When true, inherits Differentiated
+          Service Code Point (DSCP) field between inner and outer
+          header for ip6_tunnel.</para>
```

Comment:
I meant to actually put that information in the man page, along with the missing 'the'.